src/test/emulators/auth/setAccountInfo.spec.ts

```diff
@@ -415,6 +416,416 @@ describeAuthEmulator("accounts:update", ({ authApi, getClock }) => {
       });
   });
 
+  it("should allow creating MFA info", async () => {
+    const user = { email: "bob@example.com", password: "notasecret" };
+    const { localId, idToken } = await registerUser(authApi(), user);
+
+    const mfaInfo = {
+      mfaEnrollmentId: randomId(28),
+      displayName: "Cell Phone",
+      phoneInfo: TEST_PHONE_NUMBER,
+    };
+
+    await authApi()
+      .post("/identitytoolkit.googleapis.com/v1/accounts:update")
+      .set("Authorization", "Bearer owner")
+      .send({
+        localId,
+        mfa: {
+          enrollments: [mfaInfo],
+        },
+      })
+      .then((res) => expectStatusCode(200, res));
+
+    const info = await getAccountInfoByIdToken(authApi(), idToken);
+    expect(info.mfaInfo).to.have.length(1);
+    const updated = info.mfaInfo?.pop();
```

Comment:
```suggestion
    const updated = info.mfaInfo![0];
```

Don't mutate the array